Chapters/05_Monads.md

```diff
@@ -84,19 +84,27 @@ TODO[[ Use Strings everywhere instead of Char]]
 // Monads/ShowResult.scala
 
 def check(
+    // This isn't really an id. Better name?
     id: String,
     end: String,
-    msg: String
+    history: String
 ): Result =
   val result =
     if end == id then
-      Fail(msg + id)
+      Fail(history + id)
     else
-      Success(msg + id)
-  println(s"$end => check($id): $result")
```

Comment:
This didn't sit well with me even when it was a nested function, because the lambda syntax means something.
We should be careful about re-using it for display purposes.